elasticsearch7/search-index/src/main/java/org/vertexium/elasticsearch7/Elasticsearch7SearchIndex.java

```diff
@@ -373,13 +364,58 @@ void addElement(
             bulkUpdateService.flushUntilElementIdIsComplete(element.getId());
```

Comment:
We should wait in the BulkUpdateService when the batch is converted to a request.